wolox-books/src/views/Register.vue

```diff
@@ -0,0 +1,101 @@
+<template lang='pug'>
+  div.container
+      div.container-register
+        img.logo(alt="Wolox logo" src="../assets/logo.png")
+        form(v-on:click='handleSignUp')
+          .container-field(v-for='(field, name) in fields')
+            label.label(:for='name') {{field.label}}
+            input.input(v-bind:type='field.type' v-model='field.value' :id='name')
+          button.button-sign-up(type='button') Sign Up
+        hr.separator
+        button.button-login(type='submit') Login
+</template>
+<script>
+
+export default {
+  name: 'Register',
+  data () {
+    return {
+      fields: {
+        firstName: { label: 'First name', value: null },
+        lastName: { label: 'Last name', value: null },
+        email: { label: 'Email', value: null },
+        password: { label: 'Password', value: null, type: 'password' }
+      }
+    }
+  },
+  methods: {
+    handleSignUp () {
```

Comment:
Don't leave a space after the method name. If the linter is giving you a warning, override the rule